### PR TITLE
ODQA frontend + repro topk issues resolved 

### PR DESCRIPTION
**Pull request by ArthurChen189**
issue #1402 resolved

### CODE REVIEW

File: scripts/repro_matrix/run_all_odqa.py [hunk 1]
```diff
@@ -44,6 +44,8 @@ def print_results(metric, topics):
                         default=False, help='Skip running trec_eval.')
     parser.add_argument('--topics', choices=['triviaqa', 'naturalquestion'],
                         help='Topics to be run [triviaqa, naturalquestion]', required=True)
+    parser.add_argument('--full_topk', action='store_true',
```
Comment (lintool) [inline scripts/repro_matrix/run_all_odqa.py:47]: Change `--full_topk` to `--full-topk`?

More consistent with rest of code base and more readable.

**Review by lintool — CHANGES_REQUESTED** [review]
You're changing the eval... but where are the changing the part where you only retrieval top-k  from Lucene?

File: scripts/repro_matrix/run_all_odqa.py (hunk 1 above)
Comment (ArthurChen189) [reply scripts/repro_matrix/run_all_odqa.py:47]: Interesting. I thought that was a parsing issue. The dash turns out to be parsed as an underscore.

**Comment by ArthurChen189** [comment]
Oh, I thought we only want topk100 on the eval end since evaluating topk1000 requires re-retrieving.

**Comment by lintool** [comment]
> Oh, I thought we only want topk100 on the eval end since evaluating topk1000 requires re-retrieving.

Yes, correct - but that means we only need to fetch top 100 from Lucene also, right? That will make the Lucene part faster also.

**Comment by ArthurChen189** [comment]
Fixed
Notes:
- web page uses default hits = 100

**Review by lintool — APPROVED** [review]

**Pull request merged**